### ISSUE TITLE
Draft PRs should not be assigned for review

### ISSUE THREAD

**Issue by andreoss** [self-xdsd/self-core#902]
The bot should assign a reviewer after PR was marked ready for review. 

![image](https://user-images.githubusercontent.com/49783909/104200552-cfe6c600-53f6-11eb-8983-85d4f4bc700f.png)

**Comment by charlesmike** [comment]
@andreoss thank you for reporting this. I'll assign someone to take care of it soon.

**Comment by charlesmike** [comment]
@amihaiemil this is your task now, please go ahead. Deadline (when this ticket should be closed) is ``2021-01-21T17:26:16.364069``.

Estimation here is ``60 minutes``, that's how much you will be paid.

Remember, you don't have to solve everything in this ticket - you can solve it [partially](https://docs.self-xdsd.com/microtasking.html) and leave ``todo`` markers in the code, which will become future tasks.

If you have any questions don't ask me, I'm not a technical person. Open new tickets instead.

If you don't have time or simply don't want to work on this, you can always [resign](https://docs.self-xdsd.com/projectmanager.html#commands).

**Comment by amihaiemil** [comment]
@andreoss Good idea. We will do it, but a bit later. Now we are preparing the first Prod release and this change is not so straight-forward.

**Comment by amihaiemil** [comment]
@charlesmike remove

**Comment by charlesmike** [comment]
> @charlesmike remove

@amihaiemil ok, I've removed this task from scope. I'm not managing it anymore.
